vmwareengine/cloud-client/update_policy.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START vmwareengine_update_policy]
+from google.api_core import operation
+from google.cloud import vmwareengine_v1
+
+
+def update_network_policy(project_id: str, region: str) -> operation.Operation:
+    """
+    Creates a new network policy in a given network.
```

Comment:
I don't fully understand what this sample does. If I understand correctly it's turning off internet access and external_ip? If this is what it does then probably name of the function and description should be updated. Other option is to add internet_access and external_ip as arguments.

Reply:
It's going to be presented in the [Disable the internet access service in a region](https://cloud.google.com/vmware-engine/docs/networking/workload-internet-access#disable_the_internet_access_service_in_a_region)

I think I will make the internet access and external_ip parameters.

Reply:
SG.